### PR TITLE
Update to play 2.7

### LINKED ISSUE #187: Update to Play 2.7
Hello,

[Play 2.7 is here](https://blog.playframework.com/play-2-7-0-is-here/).

Could you please update your library to be compatible with this version?

Currently in [my project](https://github.com/KelkooGroup/theGardener/tree/feature/update-play) I get[ the following exception](https://travis-ci.org/KelkooGroup/theGardener/builds/487516366) `java.lang.NoClassDefFoundError: Could not initialize class io.swagger.converter.ModelConverters`

Many thanks

### PR DESCRIPTION
**Pull request by notflorian**
Fix #187

### CODE REVIEW

**Comment by loriepisicchio** [comment]
Hi! Any chance you'll merge this one soon?

**Comment by notflorian** [comment]
I'm going to update this pull request with the swagger-scala-module dependencies to 1.0.5. 
@frantuma Could you please review the pull request and merge it when this is done? Thanks :)

**Comment by notflorian** [comment]
The pull request has been rebased with master, and is now ready to be merged.

File: play-2.7/swagger-play2/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+* swagger-play2 1.2.0 support scala 2.9.x and play 2.0, please use 1.2.0.
+
+Usage
+-----
+
+You can depend on pre-built libraries in maven central by adding the following dependency:
+
+```
+libraryDependencies ++= Seq(
+  "io.swagger" %% "swagger-play2" % "1.6.1-SNAPSHOT"
```
Comment (frantuma) [inline play-2.7/swagger-play2/README.md]: can we change this to `1.7.0` ?

**Review by frantuma — COMMENTED** [review]
@notflorian Thanks! can you just update version in readme before merging?

File: play-2.7/swagger-play2/README.md (hunk 1 above)
Comment (notflorian) [reply play-2.7/swagger-play2/README.md]: Done :)

**Pull request merged**